src/components/inspector/IssuesPanel.tsx

```diff
@@ -0,0 +1,166 @@
+  transition: transform 0.2s ease;
+
+  &:hover {
+    opacity: 0.8;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  svg {
+    width: 19px;
+    height: 19px;
+  }
+`
+
+const WarningsList = styled.div<{ isVisible: boolean }>`
+  display: ${(props) => (props.isVisible ? 'block' : 'none')};
+`
+
+const WarningItem = styled.div<{ isFocused: boolean }>`
+  padding: 12px 8px;
+  margin-bottom: ${(props) => props.theme.grid.unit * 2}px;
+  border: ${(props) =>
+    props.isFocused ? `1px solid #F35143` : `1px solid #C9C9C9`};
+  box-shadow: ${(props) => (props.isFocused ? `-4px 0 0 0  #F35143` : `none`)};
+  border-radius: 4px;
+  cursor: pointer;
+  background-color: ${(props) =>
+    props.isFocused ? '#FFF1F0' + ' !important' : '#fff'};
+  position: relative;
+
+  &:hover {
+    background-color: #f2f2f2;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+`
+
+const WarningMessage = styled.p`
+  margin: 0;
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props) => props.theme.font.size.normal};
+  line-height: 1.4;
+`
+
+const NoIssuesMessage = styled.div`
+  padding: ${(props) => props.theme.grid.unit * 4}px;
+  text-align: center;
+  color: ${(props) => props.theme.colors.text.secondary};
+  font-style: italic;
+`
+
+export const IssuesPanel: React.FC = () => {
+  const [isCollapsed, setIsCollapsed] = useState(false)
+  const [selectedWarningIndex, setSelectedWarningIndex] = useState<
+    number | null
+  >(null)
+  const [store] = useStore((store) => ({
+    view: store.view,
+  }))
+
+  const warnings = store.view?.state
+    ? detectInconsistencyPluginKey.getState(store.view.state)?.warnings || []
+    : []
+
+  const handleWarningClick = (warning: Warning, index: number) => {
+    setSelectedWarningIndex(index)
+    if (store.view) {
+      const tr = store.view.state.tr
+      tr.setSelection(NodeSelection.create(tr.doc, warning.pos))
+      store.view.dispatch(tr)
+
+      const domNode = store.view.nodeDOM(warning.pos)
+      if (domNode && domNode instanceof HTMLElement) {
+        scrollIntoView(domNode)
+      }
+    }
+  }
+
+  const toggleCollapsed = () => {
+    setIsCollapsed(!isCollapsed)
+  }
+
+  return (
+    <IssuesContainer>
+      <CollapsibleHeader onClick={toggleCollapsed}>
+        <HeaderTitle>Inconsistencies ({warnings.length})</HeaderTitle>
+        <CollapseIcon isCollapsed={isCollapsed}>
+          <ArrowDownCircleIcon />
+        </CollapseIcon>
+      </CollapsibleHeader>
+
+      <WarningsList isVisible={!isCollapsed}>
+        {warnings.length === 0 ? (
+          <NoIssuesMessage>No issues found</NoIssuesMessage>
+        ) : (
+          warnings.map((warning: Warning, index: number) => (
+            <WarningItem
+              key={index}
+              isFocused={selectedWarningIndex === index}
+              onClick={() => handleWarningClick(warning, index)}
+              data-cy="warning"
+            >
+              <WarningMessage>{warning.message}</WarningMessage>
```

Comment:
generally in this implementation warning and inconsistency used interchangeably. We really should pick just one term an used it for everything. Inconsistency sounds better to me since it reflects the meaning in a more precise way than warning.